quax/_core.py

```diff
@@ -141,76 +141,51 @@ def _wrap_if_array(x: Union[ArrayLike, "Value"]) -> "Value":
 
 
 class _QuaxTrace(core.Trace[_QuaxTracer]):
-    def pure(self, val: ArrayLike) -> _QuaxTracer:
-        if _is_value(val):
-            raise TypeError(
```

Comment:
If it's important to keep this behavior, can you give me a test that exercises it?

Reply:
Sure thing! The following should raise an error:
```python
import quax
import jax

x = jax.numpy.arange(4.).reshape(2, 2)
key = jax.random.key(0)
y = quax.lora.LoraArray(x, rank=1, key=key)

def f(x):
    return jax.lax.add_p.bind(x, y)

quax.quaxify(f)(y)
```

Basically this is just a check that all of our Quax values are properly wrapped into tracers.